spec/models/dish_spec.rb

```diff
@@ -9,6 +9,8 @@
 
   describe "relationships" do
       it {should belong_to :chef}
+      it {should have_many :dish_ingredients}
+      it { should have_many(:ingredients).through(:dish_ingredients) }
   end
 
 end
```

Comment:
hmmm, I don't see a test for the Dish model method of `calorie_count`. Remember to always test your model methods.